webservices/schemas.py

```diff
@@ -941,5 +941,13 @@ class TotalsCommitteeSchema(schemas['CommitteeHistorySchema']):
 register_schema(ScheduleAByStateRecipientTotalsSchema)
 register_schema(ScheduleAByStateRecipientTotalsPageSchema)
 
+AuditFindingSchema = make_schema(
+    models.AuditFinding,
+    options={'exclude': ('idx', 'finding_id', 'tier')},
```

Comment:
Two things here:

* `idx` is not a property that exists on the `AuditFinding` model, so it wouldn't be found here.
* By setting these fields in the `exclude` option, you will prevent them from displaying in the API results, which may not be what you want!  The called to `make_schema` with just the model will expose most things by default for you. :-)

If we remove the options that exclude these fields, this will make the fields appear in the API.

Reply:
Minor correction:  We probably do still want to exclude the ID, but we'll need to rename the field to match the model property.